webapp/layouts/default.vue

```diff
@@ -21,15 +21,7 @@
               :class="{ 'hide-mobile-menu': !toggleMobileMenu }"
             >
               <div id="nav-search-box" v-if="isLoggedIn">
```

Comment:
could we get rid of this additional wrapper? it's a lot of `divs` in `divs` in `divs`... :confused:
the styles could probably be transferred to the `searchable-input` class instead